test/test_fake_tensor.py

```diff
@@ -602,6 +602,17 @@ def test_tensor_constructors_all_have_kwarg_device(self):
                 has_kwarg_device or op == torch.ops.aten._list_to_tensor.default
             )
 
+    @unittest.expectedFailure
+    def test_sparse_new(self):
+        with FakeTensorMode():
+            indices = torch.randn(1, 1, dtype=torch.int64)
+            values = torch.randn(1)
+            extra = (2,)
+            sparse = torch.randn(1).to_sparse()
+            # This used to segfault, now it does not, but it still raises an
+            # error
+            sparse2 = sparse.new(indices, values, extra)
```

Comment:
@eellison I'm not sure what the right way to make this work is, so leaving this as an xfail for now.